# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../netkit-ntalk/glibc2.2.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./talkd/announce.c.orig	Tue Oct 16 18:38:22 2001
+++ ./talkd/announce.c	Tue Oct 16 18:38:54 2001
@@ -44,6 +44,7 @@
 #include <sys/socket.h>
 #include <errno.h>
 #include <syslog.h>
+#include <time.h>
 #include <unistd.h>
 #include <stdio.h>
 #include <fcntl.h>
